qubes-artwork.spec

```diff
@@ -559,6 +559,18 @@ xdg-icon-resource forceupdate --theme oxygen || :
 %{_datadir}/plymouth/themes/qubes-dark/qubes-logo-outline.png
 %{_datadir}/plymouth/themes/qubes-dark/qubes-logo-solid.png
 
+# apple efi boot camp artwork
+/usr/share/pixmaps/bootloader/apple/qubes-install/disk_label
```

Comment:
is there some type of prefix I can use here instead of hardcoding?

Reply:
%{_datadir} instead of /usr/share could work, but I wouldn't care.